### PR TITLE
feat: add settings to Flutter App

### PR DESCRIPTION
**Pull request by sashatalalasha**
This pull request adds settings to the Flutter App. 

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: flutter-ory-network/lib/blocs/auth/auth_event.dart [hunk 1]
```diff
@@ -6,15 +6,16 @@ part of 'auth_bloc.dart';
 @immutable
 sealed class AuthEvent extends Equatable {
   @override
-  List<Object> get props => [];
+  List<Object?> get props => [];
```
Comment (jonas-jonas) [inline flutter-ory-network/lib/blocs/auth/auth_event.dart:9]: Is using a `List` some sort of convention? Maybe a record can work, too?https://dart.dev/language/records

File: flutter-ory-network/lib/pages/home.dart [hunk 2]
```diff
@@ -102,7 +110,7 @@ class _HomePageState extends State<HomePage> {
             const Padding(
                 padding: EdgeInsets.only(top: 15.0),
                 child: Text(
-                    "This app mackes REST requests to ORY Kratos' Public API to validate and decode the ORY Kratos Session payload:")),
+                    'This app mackes REST requests to ORY Kratos Public API to validate and decode the ORY Kratos Session payload:')),
```
Comment (jonas-jonas) [inline flutter-ory-network/lib/pages/home.dart]:
```suggestion
                    'This app makes REST requests to Ory Kratos Public API to validate and decode the Ory Kratos Session payload:')),
```

File: flutter-ory-network/lib/blocs/auth/auth_event.dart (hunk 1 above)
Comment (sashatalalasha) [reply flutter-ory-network/lib/blocs/auth/auth_event.dart:9]: Equatable class defines props as a function that returns a List, so it's the only valid override

**Review by jonas-jonas — APPROVED** [review]
LGTM

**Pull request merged**